README.rst

```diff
@@ -31,7 +34,33 @@ Versioning
 ----------
 
 The TUF specification uses `Semantic Versioning 2.0.0 <https://semver.org/>`_
-for its version numbers.
+(semver) for its version numbers, and a gitflow-based release management:
+
+- The 'master' branch of this repository always points to the latest stable
+  version of the specification.
+- The 'draft' branch of this repository always points to the latest development
+  version of the specification and must always be based off of the latest
+  'master' branch.
+- Contributors must submit changes as pull requests against these branches,
+  depending on the type of the change (see semver rules).
+- For patch-type changes, pull requests may be submitted directly against the
+  'master' branch.
+- For major- and minor-type changes, pull requests must be submitted against
+  the 'draft' branch.
+- Maintainers may, from time to time, decide that the 'draft' branch is ready
+  for a new major or minor release, and submit a pull request from 'draft'
+  against 'master'.
+- Before merging a branch with 'master' the 'last modified date' and 'version'
+  in the specification header must be bumped.
+- Merges with 'master' that originate from the 'draft' branch must bump either
+  the major or minor version number.
+- Merges with 'master' that originate from any other branch must bump the patch
+  version number.
+- Merges with 'master' must be followed by a git tag for the new version
+  number.
```

Comment:
This is currently not verified by `check_release.py`, because the script is executed before the merge, and the tag should probably be created after the merge.  It would be nice to automate tag or release creation.

Reply:
Any thoughts, anyone?

Reply:
I agree this would be nice to automate.

Reply:
Okay, I can look into it.